src/psyclone/tests/dynamo0p3_multigrid_test.py

```diff
@@ -829,8 +832,9 @@ def test_restrict_prolong_chain_anyd(tmpdir):
     # Now do some transformations
     otrans = DynamoOMPParallelLoopTrans()
     ctrans = Dynamo0p3ColourTrans()
-    # Apply OMP to the first restrict kernel
-    otrans.apply(schedule[0])
+    # Apply OMP to the first restrict kernel. Strictly speaking this will
+    # result in a race condition so we have to 'force' it.
+    otrans.apply(schedule[0], options={"force": True})
```

Comment:
I see you had to add a few `options={"force": True}`, do you know why did we allow this before? I thought we just allowed colour type loops or kernels to discontinuous field, what has changed in this case?

Also, if it is really a race condition, maybe you could update some of these test that already use Dynamo0p3ColourTrans, to be applied to the loop first? (although if they requires too many changes to the assert texts I will also be fine with the force option)

Reply:
Oops, I'd forgotten about these. I started out 'fixing' the tests using `force` and then decided that I should do things properly and meant to go back and undo those changes.

Reply:
OK, all of those forces should be gone now.